torch/lib/THC/THCTensorSort.cu

```diff
@@ -30,7 +30,7 @@ void THCudaLongTensor_fillSliceWithIndex(THCState* state,
       info, numSlices, sliceSize, info.strides[collapseDim])
 
   if (TensorUtils<THCudaLongTensor>::canUse32BitIndexMath(state, t)) {
-    TensorInfo<long, unsigned int> info =
+    TensorInfo<int64_t, unsigned int> info =
```

Comment:
nit: can also change `unsigned int` to `uint32_t` to be more consistent with this change in other places